en/setup/dev_env_windows_cygwin.md

```diff
@@ -1,202 +1,225 @@
 # Windows Cygwin Toolchain
 
-## Ready to use MSI Installer Download
+This is the newest (and best!) toolchain for developing PX4 on Windows.
```

Comment:
@MaEtUgR So here I put an explanation of the solution features (very high level).